tests/testthat/test-serializer-csv.R

```diff
@@ -0,0 +1,26 @@
+context("CSV serializer")
+
+test_that("CSV serializes properly", {
+  d <- data.frame(a=1, b=2, c="hi")
+  val <- serializer_csv()(d, data.frame(), PlumberResponse$new(), stop)
+  expect_equal(val$status, 200L)
+  expect_equal(val$headers$`Content-Type`, "text/plain")
+  expect_equal(val$body, readr::format_csv(d))
+
+  d <- data.frame(a=1, b=2, c="hi", na=NA)
+  val <- serializer_csv()(d, data.frame(), PlumberResponse$new(), stop)
```

Comment:
Let's add a `na = "test-na"` in `serializer_csv()` to test that extra arguments are being passed through.

Reply:
added in 3d5c882